db.js

```diff
@@ -0,0 +1,113 @@
+const Sequelize = require('sequelize');
+const {STRING, UUID, UUIDV4, DataTypes} = Sequelize;
+const conn = new Sequelize(process.env.DATABASE_URL || 'postgres://localhost/grace_shopper_db');
+
+const User = conn.define('user', {
+  id: {
+    primaryKey: true,
+    type: UUID,
+    defaultValue: UUIDV4
+  },
+  name: {
+    type: STRING,
+    allowNull: false
+  },
+  email: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+      isEmail: true
+    }
+  },
+  password: {
+    type: STRING,
+    allowNull: false
+  }
+})
+
+const Product = conn.define('product', {
+  id: {
+    primaryKey: true,
+    type: UUID,
+    defaultValue: UUIDV4
+  },
+  name: {
+    type: STRING,
+    allowNull: false
+  },
+  price: {
+    type: DataTypes.DECIMAL(10,2),
+    validate: {
+      isDecimal: true
+    }
+  }
+});
+
+Product.belongsTo(User)
```

Comment:
This whole relation is going to explode.

Reply:
You will have to sit long and hard and think about the entity relationships needed to have carts, orders, and track things like quantity.